exif.js

```diff
@@ -468,7 +472,7 @@
         }
 
         var offset = 2,
-            length = file.byteLength;
+            length = file.byteLength-5; //we will check the 5 next bytes
```

Comment:
This is unrelated change, bugfix I guess? (Just trying to understand this PR to evaluate using it in a project ;))